chain/actions/src/action_queue.rs

```diff
@@ -56,12 +56,21 @@ pub trait TransactionExecutor {
     /// Initiates closure of an outgoing channel.
     async fn initiate_outgoing_channel_closure(&self, dst: Address) -> Result<Hash>;
 
+    /// Initiates closure of multiple outgoing channels.
+    async fn initiate_multiple_outgoing_channels_closure(&self, destinations: Vec<Address>) -> Result<Hash>;
```

Comment:
Idea: Instead of introducing new methods, the original methods could've been updated to take `T: IntoIter<Address>` and internally they'd make the distinction. 

As a user of the API, it is now unclear if I can use the multicall methods with a vector of a single element.